matter/src/data_model/objects/cluster.rs

```diff
@@ -82,6 +82,42 @@ pub trait ClusterType {
     ) -> Result<(), IMStatusCode> {
         self.base_mut().write_attribute_from_tlv(attr.attr_id, data)
     }
+
```

Comment:
I was thinking of doing something similar myself, in another context. But now we have 2 read_attribute variants in the trait, `read_custom_attribute()` and `read_attribute()`.  I wonder if we can retain all the common 'checks' part of the read_attribute in a common function and allow to override the implementation just for custom-attribute/AttrVal.